packages/datagateway-dataview/src/views/card/datasetCardView.component.test.tsx

```diff
@@ -89,104 +88,90 @@ describe('Dataset - Card View', () => {
     jest.clearAllMocks();
   });
 
-  it('renders correctly', () => {
-    const wrapper = createWrapper();
-    expect(wrapper.find('CardView').props()).toMatchSnapshot();
+  // TODO: this results in a huge snapshot diff, so this is skipped for now
+  it.skip('renders correctly', () => {
+    const { asFragment } = renderComponent();
+    expect(asFragment()).toMatchSnapshot();
   });
```

Comment:
We could just change this test to test that some `Card`s are rendered as expected (e.g. with Dataset specific things)

Reply:
Yeah I am working on moving away from snapshot testing, so expect this to be redone soon.